packages/k6/README.md

```diff
@@ -139,3 +139,56 @@ The actual RPS the EA can handle will be somewhere between these two values, wit
 It's important to note that this is the RPS it can handle with the configured amount of unique requests, as a lower
 amount of unique requests can be easier to handle than a higher amount. Try out different amounts of unique requests to
 understand how the RPS limits changes.
+
+## Upper limit load testing with multiple EAs
+
+### Setup
+
+#### 1. Mock API
+
+Pull down the (private) EA tooling repo and store the path to the `mock-api` directory as the `MOCK_API_DIR` env var.
+E.g.:
+
+```bash
+git clone [...] && cd external-adapters-tooling/mock-api
+export MOCK_API_DIR=$(pwd)
+```
+
+#### 2. Configuration
+
+For each EA that you want to run, create an .env file in this directory named `[adapter].env`.
+E.g.:
+
+```dotenv
+# coingecko.env
+CACHE_ENABLED=true
+```
+
+Then, remember to set the source URL for each EA you want to run in the `[adapter]_SOURCE_URL` env var.
```

Comment:
Agreed that would be better, will rename.

Reply:
No this isn't the mock API endpoint, that's set automatically by the script. This would be the data provider API endpoint. For example, coingecko would be `https://api.coingecko.com/api/v3`.

Reply:
Sure, updated!